src/app/content/search/components/SearchResultsSidebar/RelatedKeyTerms.tsx

```diff
@@ -26,7 +26,7 @@ const RelatedKeyTerms = ({ book, keyTermHits, selectedResult }: {
       book={book}
       hits={keyTermHits}
       testId='related-key-term-result'
-      getPage={(hit) => assertDefined(findArchiveTreeNodeById(book.tree, hit.source.pageId), 'hit has to be in a book')}
+      getPage={(hit: SearchResultHit) => assertDefined(findArchiveTreeNodeById(book.tree, hit.source.pageId), 'hit has to be in a book')}
```

Comment:
That's odd that the type inference stopped working, I wonder why 🤔

Reply:
It is weird. I wonder whether having TS fully upgraded would make a difference?

Reply:
@jivey The upgrade-typescript PR is approved but not merged. I tried upgrading TS locally and taking out the type declaration, and it is still a problem. It is likely related to one of the other upgrades in this PR. The update of @types/react-redux was committed with three type declarations; it is likely the reason the declarations became an issue.